server.js

```diff
@@ -21,6 +21,8 @@ app.route('/')
   .post(function(req, res) {
     if(req.body.name.toLowerCase().indexOf('magic eight ball') < 0 && req.body.text.toLowerCase().indexOf('magic eight ball') > -1) {
       setTimeout(sayBot(res), 4000);
+    }else if(req.body.name.toLowerCase().indexOf('magic 8 ball') < 0 && req.body.text.toLowerCase().indexOf('magic 8 ball') > -1) {
```

Comment:
This else if statement does the same thing. Why not just extract the check that it is doing into a function and feed that function the different names you want to accept? That way you avoid duplication and smaller line lengths.